samples/todo-app/deployment.yaml

```diff
@@ -102,6 +102,18 @@ spec:
       containers:
       - name: stats-api
         image: azdspublic/todo-app-stats-api
+        livenessProbe:
```

Comment:
Should the other services also have probes?

Reply:
They can, but it's not necessary. Probes are optional on kubernetes pods. The idea of this is to add probes to the service that is tested with bridge.